src/Object/EMail/ItemCCEMail.php

```diff
@@ -36,20 +37,22 @@ class ItemCCEMail extends Email
 {
 	public function __construct(App $a, L10n $l10n, BaseURL $baseUrl, array $item, string $toAddress, string $authorThumb)
 	{
-		$disclaimer = '<hr />' . $l10n->t('This message was sent to you by %s, a member of the Friendica social network.', $a->user['username'])
+		$user = User::getById($a->getUserId());
+
+		$disclaimer = '<hr />' . $l10n->t('This message was sent to you by %s, a member of the Friendica social network.', $user['username'])
```

Comment:
Do we sanitize user display names for HTML?

Reply:
I don't know. I replaced much too many code lines to had got a deep look at every place :-)